src/settings.ts

```diff
@@ -35,17 +34,11 @@ export const SPARKLEVERSE_PRIVACY_POLICY =
   "https://sparklever.se/privacy-policy/";
 export const PLATFORM_BRAND_NAME = "Sparkle";
 
-export const HOMEPAGE_URL = IS_BURN
-  ? SPARKLEVERSE_HOMEPAGE_URL
-  : SPARKLE_HOMEPAGE_URL;
+export const HOMEPAGE_URL = SPARKLE_HOMEPAGE_URL;
 
-export const TERMS_AND_CONDITIONS_URL = IS_BURN
-  ? SPARKLEVERSE_TERMS_AND_CONDITIONS_URL
```

Comment:
this const SPARKLEVERSE_TERMS_AND_CONDITIONS_URL can be also removed.

Reply:
removed @ dc24c4cb9d72c54adeb66ae2ff1beeb50ebe8beb